composer.json

```diff
@@ -30,9 +30,12 @@
             "mito\\sentry\\tests\\": "tests/"
         }
     },
-    "extra": {
-        "asset-installer-paths": {
-            "bower-asset-library": "vendor/bower"
+    "config": {
+        "fxp-asset": {
```

Comment:
Hello @albertborsos!
It is worth remembering that the Yii2 officially abandoned the fxp-asset.
https://github.com/yiisoft/yii2-app-advanced/blob/master/composer.json#L35